python/cog/response.py

```diff
@@ -8,6 +8,7 @@ class Status(str, enum.Enum):
     PROCESSING = "processing"
     SUCCEEDED = "succeeded"
     FAILED = "failed"
+    CANCELED = "canceled"
```

Comment:
It's been a todo for a while to use these objects in web. Now it's in Go too, perhaps we have a shared OpenAPI spec somewhere.